app/main.py

```diff
@@ -1,8 +1,46 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            comfort_class: int = 7,
+            clean_mark: int = 1,
+            brand: str = "some_car"
+    ) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
```

Comment:
In the requirements these attributes can accept only specific values. Let's check them and if value is incorrect, set some default correct value